src/main/java/uk/gov/hmcts/reform/hmc/model/hmi/HmiCaseDetails.java

```diff
@@ -34,4 +34,12 @@
 
     private Boolean caseRestrictedFlag;
 
+    private Integer caseVersionId;
+
+    private CaseLinks[] caseLinks;
```

Comment:
```suggestion
    private List<CaseLinks> caseLinks;
```
Might save some of the converting between lists and arrays that is done elsewhere.